site/app/controllers/UserProfileController.php

```diff
@@ -149,6 +149,9 @@ public function changeProfilePhoto() {
             // Save image for user
             $result = $user->setDisplayImage($extension, $_FILES['user_image']['tmp_name']);
             $display_image = $user->getDisplayImage();
+            if ($result === 2) {
```

Comment:
If I was just looking at this code without seeing the rest of the code in the PR I would have no idea what 2 means. Maybe add some constants in the User model. So instead of 2 you could say User::PROFILE_PIC_QUOTA_EXHAUSTED for example. It's of course fine for the value of the constant to be an int but it provides more clarity when looking at the code.

If you look at the top of User.php you'll see some examples of constants currently used.